core/src/avm2/op.rs

```diff
@@ -215,8 +215,7 @@ pub enum Op<'gc> {
         num_args: u32,
     },
     NewCatch {
-        #[collect(require_static)]
-        index: Index<Exception>,
```

Comment:
Personally I still like `Index`/`u32` in ops where we know it's gonna be u32 :/ But not blocking the change either.